doc/mesh-api.txt

```diff
@@ -118,6 +118,40 @@ Methods:
 			org.bluez.mesh.Error.AlreadyExists,
 			org.bluez.mesh.Error.Failed
 
+    object node, array{byte, array{(uint16, dict)}} configuration, fd
+            Attach1(object app_root, uint64 token)
```

Comment:
I don't like the name. We're not on windows ;P

Reply:
Me either but I don't have better name. Do you think that `AcquireAttach` is better?

Reply:
AttachFD, AttachStream, AttachSocket?